blazor/datagrid/batch-editing.md

```diff
@@ -338,4 +338,72 @@ Also, the add operation is handled while performing batch editing using the bool
     }
 }
 
-```
\ No newline at end of file
+```
+
+## Select entire text of cell while editing in batch editing mode
```

Comment:
Select entire text of cell while editing in batch edit mode - please use this title